manifests/params.pp

```diff
@@ -19,6 +19,9 @@
   $rvm_ruby = '2.5'
   $rvm_branch = 'stable'
 
+  $use_scl_ruby = false
```

Comment:
just one question, should we default to true? Seems like the plan is to move the dev box to the ruby SCL rather than RVM

Reply:
I'm thinking about this. How would the migration path be?

Reply:
I'd say there's no migration path. It's a devel box, so I don't feel too bad about that. If we default to true, I think we just need to post to the community forum to show people how to keep RVM turned on for existing devel boxes.

Reply:
A reinstall is also a migration path ;) But a post to the forum is good. We will get questions about this and it's easy to refer to on IRC.

Reply:
ya, if you have a dev box with RVM that is fine. RVM is mostly an issue when provisioning boxes, once they are functional things are generally fine. I wouldn't worry too much about existing boxes (unless I'm missing something).

So we should set the SCL to true here? :pray: